ui/src/app/utils/index.ts

```diff
@@ -1,5 +1,8 @@
 import {Router} from '@angular/router';
 import {fromJS} from 'immutable';
+import {
+  find
```

Comment:
Do we really need lodash find? I would generally favor using standard Javascript APIs (i.e. `[].find()` in this case) unless there's a major deficiency there.

Reply:
The builtin function would be totally suitable in this case. The `lodash/fp` versions do come with some niceties (e.g. predicate shorthands, auto-currying) that can be leveraged to make very compact, expressive code in some cases.